src/panels/lovelace/cards/hui-shopping-list-card.ts

```diff
@@ -331,9 +336,11 @@ class HuiShoppingListCard extends SubscribeMixin(LitElement)
         );
         this._renderEmptySortable = true;
         await this.updateComplete;
-        const container = this.shadowRoot!.getElementById("sortable")!;
-        while (container.lastElementChild) {
-          container.removeChild(container.lastElementChild);
+        const container = this._sortableEl;
```

Comment:
why create this?

Reply:
> why create this?

Because this element will be used multiple times below, I thought I should use a local variable to hold it, so that I won't be calling the `@query` getter too many times without knowing how expensive the `@query` getter is. It's more like an instinct, not really a solid reason.

But I change it to what you suggested, it's obviously cleaner. And since you suggested it, what I was thinking is probably unnecessary.

Reply:
Its also cached now which is very nice.